js/browser_action.js

```diff
@@ -23,10 +24,15 @@ function updatePortfolio(){
 						return obj2['holdingValueFiat'] - obj1['holdingValueFiat'];
 					}),
 					function( index, value ) {
+					var price_html = items.fiatMode ? (
+						value['fiatSymbol'] + value['lastPriceFiat'].toFixed(5) + '<br/><span class="' + value['arrow'] +'">' + value['twentyFourHourPercentChangeFiat'].toFixed(2) +'%</span>'
+					) : (
+						value['symbol'] + value['lastPrice'].toFixed(5) + '<br/><span class="' + value['arrow'] +'">' + value['twentyFourHourPercentChange'].toFixed(2) +'%</span>'

```

Comment:
In a hindsight, maybe this price should be rounded to 7 or 8 digits, as for some pairs 5 digits is too big a number
![image](https://user-images.githubusercontent.com/20352412/35669519-63685a2a-0746-11e8-97ed-ab92f749d50b.png)